flexmeasures/data/schemas/scheduling/__init__.py

```diff
@@ -464,3 +464,108 @@ def check_flex_model_sensors(self, data, **kwargs):
                 )
             sensors.append(sensor)
         return data
+
+
+class FlexModelSchema(Schema):
```

Comment:
`class DBFlexModelSchema(FlexModelSchema):`

Please note that the `FlexModelSchema` lives in `data/schemas/scheduling/storage.py`, and `DBFlexModelSchema` should live there, too.

Once you subclass from `FlexModelSchema`, there should be no need to redefine its fields, but only to add validators that forbid timeseries specs, just like what we did in `DBFlexContextSchema.forbid_time_series_specs`.